app/openai/mcp_client.py

```
@@ -0,0 +1,145 @@
+                return result.content[0].text if result.content else None
+            except Exception as e:
+                print(f"Error fetching all pets: {e}")
+                return None
+
+    async def search_pets_by_name(self, name: str):
+        """이름으로 펫 검색"""
+        async with self.create_session() as session:
+            try:
+                result = await session.call_tool(
+                    "execute_sql",
+                    {"query": f"SELECT * FROM pet WHERE name LIKE '%{name}%'"}
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)